src/ContainerFactory.ts

```diff
@@ -3,12 +3,8 @@ import { RedisClient, createClient } from 'redis';
 import { buildProviderModule } from 'inversify-binding-decorators';
 
 const createRedisClient = () => {
-    return createClient(
-        {
-            host: process.env.REDIS_HOST,
-            port: Number(process.env.REDIS_HOST)
-        }
-    ).on('error', (err) => { throw err; });
+    const redisUrl = 'redis://' + process.env.CACHE_SERVER || 'localhost:6379';
```

Comment:
No default please. We expect to fail is config is missing.

Reply:
Not a big deal but why not to use string interpolation instead of '+'?

Reply:
Good point, fixed those